text/0000-interrupt-calling-conventions.md

```diff
@@ -78,10 +78,12 @@ The following interrupt ABIs are currently supported:
 - _(unstable)_ `extern "x86-interrupt"`: This calling convention can be used for defining interrupt handlers on 32-bit and 64-bit `x86` targets. Functions must have one of the following two signatures, depending on the interrupt vector:
 
   ```rust
-  extern "x86-interrupt" fn(stack_frame: &ExceptionStackFrame);
-  extern "x86-interrupt" fn(stack_frame: &ExceptionStackFrame, error_code: u64);
+  extern "x86-interrupt" fn(stack_frame: &StackFrame);
+  extern "x86-interrupt" fn(stack_frame: &StackFrame, error_code: ErrorCode);
```

Comment:
This needs to specify what exactly is being saved on the stack, and how the interrupt is exited.

x86 supports various different asynchronous interrupts, and based on the description, it's not clear which one is supported by these calling conventions.

Reply:
These calling conventions are listed only as an example. So I think it makes more sense to specify such details elsewhere, e.g. in the future stabilization proposals for the respective calling convention.